site/_wiki/Documentation/ConfigurationGuide.md

```diff
@@ -0,0 +1,418 @@
+
+### Unloading drivers on Linux
+
+If you've installed OpenTabletDriver from an official package, the stock tablet
+drivers are already blacklisted.
+
+Still, run the following command to ensure that the upstream drivers are unloaded:
+
+```sh
+sudo rmmod hid_uclogic wacom 2>/dev/null
+```
+
+You likely need to set up udev rules so that your user can access the raw
+device directly.
+
+Normally, this is autogenerated by the build process and uses the officially
+supported configs.
+
+You can define these manually for now, and put them in a local udev rule file,
+e.g. `/etc/udev/rules.d/91-opentabletdriver-adds.rules`:
+
+```conf
+KERNEL=="hidraw*", ATTRS{idVendor}=="b33f", ATTRS{idProduct}=="c4k3", TAG+="uaccess", TAG+="udev-acl"
+SUBSYSTEM=="usb", ATTRS{idVendor}=="b33f", ATTRS{idProduct}=="c4k3", TAG+="uaccess", TAG+="udev-acl"
+```
+
+Make sure to change the values of `idVendor` and `idProduct` as appropriate.
+You can get them with `lsusb`.
+
+> Note: `idVendor` and `idProduct` must be in base-16 (hexadecimal) as opposed
+  to the base-10 format used in OpenTabletDriver configurations.
+
+See `man 7 udev` for systemd udev syntax. If you're not using systemd udev, see
+your distro documentation, or change the appropriate usb and hidraw permissions
+manually.
+
+### MacOS
+
+TODO, probably just uninstall existing drivers?
+
+## Configuration Template
+
+Getting a configuration template is usually simple if OpenTabletDriver supports
+other tablets from your manufacturer, especially if it's in the same line of
+models.
+
+- Copy a "neighbouring" configuration from the
+  `OpenTabletDriver.Configurations/Configurations/` folder
+- Rename the file to be `<model name>.json`, e.g. `H420.json`
+
+If your manufacturer does not have an existing template, please use the
+[Configuration skeleton] or copy any other tablet configuration and follow the
+guide carefully.
+
+> Note that the configuration format between OpenTabletDriver 0.6 (`0.6.x`) and
+  0.7 (`master`) differs slightly. Make sure that you pick a configuration of
+  the same format as the version of OpenTabletDriver you're using.
+
+[Configuration skeleton]: https://github.com/OpenTabletDriver/OpenTabletDriver/issues/3249
+
+## Setting specifications according to official manufacturer specifications
+
+Setting up some of the specifications now can simplify touching up later values
+if you know what they are. You can also add them as you discover the feature;
+we will ensure the specifications are set correctly at the end of the guide.
+
+Some (almost) always correct values include:
+- Pen and tablet (aux) button counts
+- Tool support (such as tablet mouse)
+- Touch support
+- Pen pressure range
+
+Digitizer dimensions (for both pen and touch digitizer) will be determined as a
+later part of the guide, and is not always correct.
+
+Note that wheels (rings) are not yet supported, and should not be directly defined
+(even as a button). See [OpenTabletDriver#1367] for more information
```

Comment:
I don't think this is clear enough that auxiliary buttons at the center of the wheel are fine to treat as normal auxiliary keys (as they are usually normal auxiliary keys)